runtimes/python-3.8/README.md

```diff
@@ -84,18 +83,17 @@ You can respond with `json()` by providing object:
 
 ```python
 def main(req, res):
-    res.json({
+    return res.json({
```

Comment:
Do we need a return here ?

Reply:
Yes. I was doing QA and function was failing. Doing `return` fixed it. Not sure why.
We do the same in tests and examples, so I think it's safe to also provide it in example.

Reply:
okay 👍